src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -290,6 +290,15 @@
   <data name="ProvideCorrectArgumentsToFormattingMethodsMessage" xml:space="preserve">
     <value>Provide correct arguments to formatting methods</value>
   </data>
+  <data name="ProvideCorrectArgumentsToFormattingMethodsInvalidFormatTitle" xml:space="preserve">
+    <value>Provide correct format specification to formatting methods</value>
+  </data>
+  <data name="ProvideCorrectArgumentsToFormattingMethodsInvalidFormatDescription" xml:space="preserve">
+    <value>The format argument that is passed to System.String.Format is not properly formatted.</value>
+  </data>
```

Comment:
Let's not add a Title and description, different title and description is not supported for same ID. I know their meaning not quite match with the suggestion, we can create an issue for deciding if we add a new ID for this and resolve in 9.0
```suggestion
```

Reply:
Done. Needed to do this locally so the "other language" resx files are updated as well.

Reply:
Thanks!